src/App.js

```diff
@@ -1,5 +1,32 @@
+import CarRace from './Model/CarRace.js';
+import InputView from './View/InputView.js';
+import OutputView from './View/OutputView.js';
+
 class App {
-  async run() {}
+  #input_view = new InputView();
+
+  #output_view = new OutputView();
+
+  async run() {
+    const carNames = await this.#input_view.readCarNamesFromInput();
+    const raceCount = await this.#input_view.readRaceCountFromInput();
+
+    const carRace = new CarRace(carNames, raceCount);
+
+    this.#output_view.printExecutionResult();
+
+    Array.from({ length: Number(raceCount) }).forEach(() => {
+      const cars = carRace.race();
+      cars.forEach((car) => {
+        this.#output_view.printRaceResult(car.getName(), car.getMoveCount());
+      });
+      this.#output_view.printLineBreak();
+    });
```

Comment:
갠적인 건데, `Array.from({ length: Number(raceCount) })` 대신 for문을 쓰는 건 어떤가 싶어요!
Array.from은 배열을 생성하는 목적이 있다고 생각하는데, for문은 정해진 횟수만큼 반복하는 목적에 가까운 것 같았어요. 또한 for문이 가독성 측면에서 낫다고 생각했어요.

+Array.from에 forEach도 좋지만 인자의 map 콜백함수를 이용하는 건 어떨까요?
```suggestion
    Array.from({ length: Number(raceCount) }, () => {
      const cars = carRace.race();
      cars.forEach((car) => {
        this.#output_view.printRaceResult(car.getName(), car.getMoveCount());
      });
      this.#output_view.printLineBreak();
    });
```

Reply:
Airbnb JS 스타일 가이드에서 for 루프보다 고차 함수 사용을 권장하는 내용이 있어서 `Array.from()`으로 배열을 생성해서 순회하였던 것인데 말씀하신 것처럼 단순히 정해진 횟수만큼 반복할 때는 for 루프를 사용하는 것도 괜찮은 것 같네요!

`Array.from()`의 두 번째 인자인 map 함수를 사용하는 것은 생각하지 못했었는데 추가적인 배열 생성 없이 반복할 수 있어서 좋은 것 같네요👍

Reply:
아하 글쿤요! 다시 생각해보니 위 코드 상황에서는 `[...arr].map()`이 아닌 `[...arr].forEach()`라 배열 하나 생성하는 건 똑같을 거라고 생각합니당. 그저 인자의 map 기능을 리턴문 없이 사용하는 건데, 어쩌면 map이라 리턴문이 없어서 다른 사람이 읽기에 헷갈릴 수도 있을 것 같네요..!